app/components/back-office-app-services/applications-migration-function/iam.tf

```diff
@@ -9,3 +9,16 @@ resource "azurerm_key_vault_access_policy" "read_secrets" {
   secret_permissions      = ["Get"]
   storage_permissions     = []
 }
+
+# Annoyingly it doesn't look like you can get a data "azurerm_storage_container" resource in a different Subscription and RG
+# So we need to build the resource ID dynamically
+resource "azurerm_role_assignment" "read_data_lake_storage" {
+  # We're only deploying the synapse migration functionality on Dev right now
+  count = var.odw_subscription_id != "" ? 1 : 0
+
+  scope                = "/subscriptions/${var.odw_subscription_id}/resourceGroups/${var.odw_resource_group_name}/providers/Microsoft.Storage/storageAccounts/${var.odw_data_lake_storage_account_name}/blobServices/default/containers/odw_curated"
```

Comment:
Might be clearer with a `join('/', ...)` but not a big deal.

Reply:
Agreed, didn't know this function existed! Cheers!